src/engine/Util/Detector.ts

```diff
@@ -1,13 +1,134 @@
 /// <reference path="Log.ts" />
+
+/**
+ * This is the list of features that will be used to log the supported
+ * features to the console when Detector.logBrowserFeatures() is called.
+ */
+const REPORTED_FEATURES: Object = {
+   webgl: 'WebGL',
+   webaudio: 'WebAudio',
+   gamepadapi: 'Gamepad API'
+};
+
 module ex {
 
    /**
+    * Interface for detected browser features matrix
+    */
+   export interface IDetectedFeatures {
+      canvas: boolean;
```

Comment:
Interesting, we updated our TS version to 2.0, if you're using VS Code what does it say in the bottom right? It should say TypeScript 2.0.10 or something... did you do an `npm install`?